cmd/delete.go

```diff
@@ -152,9 +168,37 @@ func (c deleteConfig) Prompt() (deleteConfig, error) {
 
 	dc := deleteConfig{}
 
-	return dc, survey.AskOne(
-		&survey.Input{
-			Message: "Function to remove:",
-			Default: deriveName(c.Name, c.Path)},
-		&dc.Name, survey.WithValidator(survey.Required))
+	var qs = []*survey.Question{
+		{
+			Name: "name",
+			Prompt: &survey.Input{
+				Message: "Function to remove:",
+				Default: deriveName(c.Name, c.Path)},
+			Validate: survey.Required,
+		},
+		{
+			Name: "all",
+			Prompt: &survey.Confirm{
+				Message: "Do you want to delete all resources?",
+				Default: true,
```

Comment:
```
Default: c.DeleteAll,
```
This _question_ default could be set to the current value of the config object's default.  That way if the user passes in the flag `--all=false`, but also `--confirm`, the value of the question will be set to the value they specified on the command line first.